project/Dependencies.scala

```diff
@@ -440,8 +440,8 @@ object Dependencies {
 
   val UnixDomainSocket = Seq(
     libraryDependencies ++= Seq(
-        "com.github.jnr" % "jffi" % "1.2.17" classifier "complete", // ApacheV2
```

Comment:
Unsure if the classifier is still required. It certainly used to be. What’s the downside of having it still?

Reply:
My immediate problem is that it (with Coursier) breaks the Whitesource reporting.

Reply:
Coursier has problems with artifacts, see e.g. https://github.com/sbt/sbt/issues/5325 (and the linked issues there as well).

Reply:
The testsuite in this branch runs fine for me locally.